packages/terra-status-view/src/StatusView.jsx

```diff
@@ -11,22 +13,27 @@ const StatusViewVariants = {
   NOMATCHINGRESULTS: 'no-matching-results',
   NOTAUTHORIZED: 'not-authorized',
   ERROR: 'error',
-  CUSTOM: 'custom',
 };
 
 const propTypes = {
   /**
-   * An array of actionable buttons to display.
+   * An array of objects containg terra-button properties. Use these to render buttons in the bottom section.
+   * Example:`[{ text: 'Button 1', key: 1, size: 'medium', variant: 'action', onClick: onClickFunction}]`
    */
-  children: PropTypes.node,
+  buttonAttrs: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Can we use Button.propType instead of PropTypes.object?

Reply:
updated - i had to wrap that in a `shape`, otherwise react complains